tests/test_examples.py

```diff
@@ -177,10 +177,14 @@ def test_svm_batch(self):
     ('cholesky_batch', 'cholesky_batch.csv', 'choleskyFactor'),
     ('covariance_batch', 'covariance.csv', 'covariance'),
     ('covariance_streaming', 'covariance.csv', 'covariance'),
-    ('decision_forest_classification_batch', None, lambda r: r[1].prediction,
-     (2019, 'P', 1)),
-    ('decision_forest_regression_batch', 'decision_forest_regression_batch.csv',
-     lambda r: r[1].prediction, (2019, 1)),
+    ('decision_forest_classification_default_dense_batch',
+     None, lambda r: r[1].prediction, (2019, 'P', 1)),
```

Comment:
You can remove old checks (less than 2020.3)

Reply:
Thanks, old versions were removed, information about hist method was added with [appropriate PR](https://github.com/oneapi-src/oneDAL/pull/1217)